frontend/vue/components/Statevector/StatevectorBrackets.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="statevector-brackets">
+    <slot />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue-demi'
+
+export default defineComponent({ name: 'StatevectorBrackets' })
+</script>
+
+<style lang="scss" scoped>
+@import 'carbon-components/scss/globals/scss/typography';
```

Comment:
Do we need this import?

```suggestion
```

Reply:
I think it adds spacing variables indirectly. Maybe I can replace it with the layout lib.

Reply:
If needed,  `carbon-components/scss/globals/scss/spacing` is also available.